### ISSUE TITLE
Implement Dropout in Model

### ISSUE THREAD

**Issue by SrujanaBotcha** [B-Srujana/IS640-GPT-Project#19]
Objective:
Update the provided model by incorporating Dropout layers to regularize the neural network and prevent overfitting. This milestone will help improve the generalization of the model during training.

**Comment by SrujanaBotcha** [comment]
Updated the code for Milestone8. Met with the requirements.
